### PR TITLE
[WIP] New tooltip for wasta with new model, mako and translations

### PR DESCRIPTION
**Pull request by alcapat**

### CODE REVIEW

File: chsdi/locale/fi/LC_MESSAGES/chsdi.po [hunk 1]
```diff
@@ -8766,6 +8775,18 @@ msgstr "Standort"
 msgid "tt_ch.bfe.statistik-wasserkraftanlagen_name"
 msgstr "Name"
 
+msgid "tt_ch.bfe.statistik-wasserkraftanlagen_performancegeneratormaximum"
```
Comment (loicgasser) [inline chsdi/locale/fi/LC_MESSAGES/chsdi.po]: please follow the convention.
ch.bfe.statistik-wasserkraftanlagen.performancegeneratormaximum

and you don't need to add the translations it is automatic when you fill up, ms_item.

File: chsdi/models/vector/uvek.py [hunk 2]
```diff
@@ -632,6 +632,37 @@ class MinergieGebaeude(Base, Vector):
...
+    __table_args__ = ({'schema': 'bfe', 'autoload': False})
+    __template__ = 'templates/htmlpopup/statistikwasserkraftanlagen_new.mako'
+    __bodId__ = 'ch.bfe.statistik-wasserkraftanlagen'
+    __extended_info__ = True
+    __label__ = 'name'
+    id = Column('wastanumber', Integer, primary_key=True)
+    name = Column('name', Text)
+    location = Column('location', Text)
+    canton = Column('canton', Text)
+    hydropowerplantoperationalstatus_it = Column('hydropowerplantoperationalstatus_it', Text)
```
Comment (loicgasser) [inline chsdi/models/vector/uvek.py]: Unicode everywhere you used text.

File: chsdi/models/vector/uvek.py [hunk 3]
```diff
@@ -632,6 +632,37 @@ class MinergieGebaeude(Base, Vector):
...
+    hydropowerplanttype_it = Column('hydropowerplanttype_it', Text)
+    hydropowerplantoperationalstatus_fr = Column('hydropowerplantoperationalstatus_fr', Text)
+    hydropowerplanttype_fr = Column('hydropowerplanttype_fr', Text)
+    hydropowerplantoperationalstatus_de = Column('hydropowerplantoperationalstatus_de', Text)
+    hydropowerplanttype_de = Column('hydropowerplanttype_de', Text)
+    beginningofoperation = Column('beginningofoperation', Integer)
+    endofoperation = Column('endofoperation', Integer)
+    dateofstatistic = Column('dateofstatistic', Text)
+    performanceturbinemaximum = Column('performanceturbinemaximum', Numeric)
```
Comment (loicgasser) [inline chsdi/models/vector/uvek.py]: Float everywhere you used Numeric (or integer if needed)

File: chsdi/models/vector/uvek.py [hunk 4]
```diff
@@ -632,6 +632,37 @@ class MinergieGebaeude(Base, Vector):
 register(MinergieGebaeude.__bodId__, MinergieGebaeude)
 
 
+class STATISTIKWASSERKRAFTANLAGEN_NEW(Base, Vector):
```
Comment (loicgasser) [inline chsdi/models/vector/uvek.py]: Convention is as follow for class names
```class StatistikwasserkraftanlagenNew```

File: chsdi/templates/htmlpopup/statistikwasserkraftanlagen_new.mako [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+<%def name="extended_info(c, lang)">
+<%
+    c['stable_id'] = True
+    lang = 'de' if lang in ('de', 'rm', 'en') else lang
+    import urllib2
+    has_picture = True
+    image_file = None
+    try:
+        image_file == urllib2.urlopen("http://www.bfe-gis.admin.ch/bilder/ch.bfe.statistik-wasserkraftanlagen/%d.jpg" % c['featureId'])
```
Comment (loicgasser) [inline chsdi/templates/htmlpopup/statistikwasserkraftanlagen_new.mako]: use requests module instead, we want to get rid of urlib2

Comment (loicgasser) [inline chsdi/templates/htmlpopup/statistikwasserkraftanlagen_new.mako]: actually if you're just testing if the image exists.
use:
h.resource_exists(url)

it uses the requests module and performs a head request to determine the existance of the image.

Implementation can be found in helpers

**Comment by AFoletti** [comment]
why are there two PR for the same layer? If this one is outdated, please close it

**Pull request closed without merging**